lambdaclass/mirra_backend
# Fix dead units targeting in delayed effects

Fixes the targeting of dead units in effects with delays

# To test

Change create_effect_map to
```
defp create_effect_map(%Effect{} = effect, skill_id),
    do: %{
      type: effect.type,
      delay: 15,
...
```

hardcoding a high delay value. This will make it so that sometimes units miss because their target died before the effect's delay ends. Battle should still work fine
